### PR TITLE
Telemetry update

### PR DESCRIPTION
**Pull request by HThuren**
Telemetry updated for CRSF info

### CODE REVIEW

File: docs/wiki/guides/current/Telemetry.md [hunk 1]
```diff
@@ -1,10 +1,53 @@
...
+| Sats	    | GPS Satellites acquired                               | raw       | GPS_ID         |   5   |
+| RxBt	    | Battery voltage                                       | V	        | BATTERY_ID     |   0   |
+| Curr	    | Current draw                                          | A         | BATTERY_ID     |   1   |
+| Capa	    | Current consumption                                   | mAh       | BATTERY_ID     |   2   |
+| Bat%	    | Battery remaining                                     | %         | BATTERY_ID     |   3   |
+| Ptch	    | FC pitch angle                                        | radians   | ATTITUDE_ID    |   0   | 
+| Roll	    | FC roll angle                                         | radians   | ATTITUDE_ID    |   1   |
+| Yaw	    | FC yaw angle                                          | radians   | ATTITUDE_ID    |   2   |
+| FM	    | Flight mode                                           | See table | FLIGHT_MODE_ID |   0   |
+|           |                                                       | below     |                |       |
```
Comment (haslinghuis) [inline docs/wiki/guides/current/Telemetry.md:33]: image

File: docs/wiki/guides/current/Telemetry.md [hunk 2]
```diff
@@ -1,10 +1,53 @@
...
+| ----------- | ------------------- |
+| !FS         | Failsafe mode       |
+| RTH         | Return To Home mode |
+| MANU        | Passthru mode       |
+| ACRO        | ACRO mode           |
+| STAB        | Angle mode          |
+| HOR         | Horizon mode        |
+| AIR         | Air mode            |
+| WAIT        | Wait for GPS lock   |
+| appended *  | FC are not ARMED    |
```
Comment (haslinghuis) [inline docs/wiki/guides/current/Telemetry.md:46]:
```suggestion
| appended *  | FC is not ARMED     |
```

Where is the * pointing at :)

File: docs/wiki/guides/current/Telemetry.md (hunk 1 above)
Comment (HThuren) [reply docs/wiki/guides/current/Telemetry.md:33]: yes, but how to have rows with more lines ?

Comment (haslinghuis) [reply docs/wiki/guides/current/Telemetry.md]: Perhaps remove `below` and maybe make `See table` a link to that table

Comment (HThuren) [reply docs/wiki/guides/current/Telemetry.md]: Good idea with link, but I haven't figured out to make links on same page in this format ?

Comment (HThuren) [reply docs/wiki/guides/current/Telemetry.md]: in fact - what are the format in use ;-)

**Comment by HThuren** [comment]
@haslinghuis great, now with links, in the end very easy .-)

Page ready to go

File: docs/wiki/guides/current/Telemetry.md [hunk 3]
```diff
@@ -1,10 +1,52 @@
...
+| RxBt	    | Battery voltage                                       | V	        | BATTERY_ID     |   0   |
+| Curr	    | Current draw                                          | A         | BATTERY_ID     |   1   |
+| Capa	    | Current consumption                                   | mAh       | BATTERY_ID     |   2   |
+| Bat%	    | Battery remaining                                     | %         | BATTERY_ID     |   3   |
+| Ptch	    | FC pitch angle                                        | radians   | ATTITUDE_ID    |   0   | 
+| Roll	    | FC roll angle                                         | radians   | ATTITUDE_ID    |   1   |
+| Yaw	    | FC yaw angle                                          | radians   | ATTITUDE_ID    |   2   |
+| FM	    | Flight mode                                           | [See below](#CRSF-flightmode) | FLIGHT_MODE_ID |   0   |
+
+#### <a id="CRSF-flightmode"></a> CRSF flightmode
```
Comment (haslinghuis) [inline docs/wiki/guides/current/Telemetry.md:34]: Looks all good now - only have to check this - as we should not need html code :thinking:

**Review by ricardocarva — APPROVED** [review]

File: docs/wiki/guides/current/Telemetry.md (hunk 3 above)
Comment (ricardocarva) [reply docs/wiki/guides/current/Telemetry.md:34]: Ditto

Comment (haslinghuis) [inline docs/wiki/guides/current/Telemetry.md:34]:
```suggestion
| FM	    | Flight mode                                           | [See below](#crsf-flightmode) | FLIGHT_MODE_ID |   0   |

#### CRSF flightmode
```

**Review by haslinghuis — CHANGES_REQUESTED** [review]
This will remove the need for html

**Review by haslinghuis — APPROVED** [review]

**Review by ricardocarva — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**